browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -136,6 +152,16 @@ void BraveToolbarView::OnThemeChanged() {
     bookmark_->UpdateImage();
 }
 
+void BraveToolbarView::OnProfileAdded(const base::FilePath& profile_path) {
```

Comment:
Is this callback only called only when new **normal** profile is created?

Reply:
@simonhong I don't see this getting called when starting / opening a guest or tor profile. Are there any other non-normal profiles I should check?

Reply:
If we don't need to check about tor/guest profile add/delete, I think current impl is fine.
FYI, we can get all profile creation by `NOTIFICATION_PROFILE_XXX` notifications.